CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file. The format
 
 > **Note**: Changes and additions listed here are contained in the `:edge` image tag. These changes may not be as stable as released changes.
 
+- **Internal:**
+  - Add warning, when update-check is enabled, but no stable release image is used ([#3684](https://github.com/docker-mailserver/docker-mailserver/pull/3684))
+
```

Comment:
Should be moved into the "Added" section?

Reply:
You literally beat me to this :D

Reply:
I will adjust and then we can merge.

Reply:
Done.